hpe3par_sdk/client.py

```diff
@@ -3680,3 +3725,106 @@ def removeVolumesFromVolumeSet(self, name, setmembers):
         self.client.modifyVolumeSet(
             name, action=client.HPE3ParClient.SET_MEM_REMOVE,
             setmembers=setmembers)
+
+    def remoteCopyGroupExists(self, name):
+        try:
+            self.getRemoteCopyGroup(name)
+        except exceptions.HTTPNotFound:
+            return False
+        return True
+
+    def remoteCopyGroupVolumeExists(self, remote_copy_group_name, volume_name):
+        try:
+            self.getRemoteCopyGroupVolume(remote_copy_group_name, volume_name)
+        except exceptions.HTTPNotFound:
+            return False
+        return True
+
+    def admitRemoteCopyLinks(self, targetName, source_port, target_port_wwn_or_ip):
```

Comment:
Please maintain the same naming convention for the variables of the method throughout the PR.
Rename source_port and target_port_wwn_or_ip to naming convention followed by targetName variable.

Do this for all the other places in this PR.